drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -307,12 +307,15 @@ def brief_name
   # Run if we changed name/DOB/SSN
   private def warehouse_match_existing_clients
     return unless warehouse_columns_changed?
+    return if Delayed::Job.where(failed_at: nil, locked_at: nil).jobs_for_class('GrdaWarehouse::Tasks::IdentifyDuplicates').jobs_for_class('match_existing!').exists?
```

Comment:
It would be nice if the base job class had a generic version of this. Like `GrdaWarehouse::Tasks::IdentifyDuplicates.queued?(method: 'match_existing!')`

Reply:
Yeah, I was also thinking of making a `pending` or `unstarted` or `queued` scope on delayed job.